lib/hook/consumers/github.ex

```diff
@@ -0,0 +1,140 @@
+defmodule Accent.Hook.Consumers.GitHub do
+  @moduledoc """
+  From a project’s integration association and a webhook event from GitHub,
+  sync and add translation as if the operations were made manually by a user.
+
+  """
+  use Accent.Hook.EventConsumer, subscribe_to: [Accent.Hook.Producers.GitHub]
+
+  alias Accent.{Document, Repo, Version}
+  alias Accent.Hook.Consumers.GitHub.AddTranslations
+  alias Accent.Hook.Consumers.GitHub.Sync
+  alias Accent.Hook.Context
+  alias Accent.Plugs.MovementContextParser
+  alias Accent.Scopes.Document, as: DocumentScope
+  alias Accent.Scopes.Version, as: VersionScope
+
+  def handle_events(events, _from, state) do
+    Enum.each(events, &handle_event/1)
+
+    {:noreply, [], state}
+  end
+
+  defp handle_event(%Context{user: user, project: project, event: "push", payload: payload}) do
```

Comment:
Would it be a good idea to add an implementation of `handle_event` for all other type of event, if any?